apps/browser-extension-wallet/src/lib/scripts/background/inject.ts

```diff
@@ -29,7 +38,23 @@ const initializeInjectedScript = (props: WalletProperties, { logger }: cip30.Ini
     }
   });
 
+  // Wrap the proxi API object with a regular javascript object to avoid interop issues with some dApps.
+  const api: WalletApi = {
+    getNetworkId: () => proxyApi.getNetworkId(),
+    getUtxos: (amount?: Cbor, paginate?: Paginate) => proxyApi.getUtxos(amount, paginate),
+    getBalance: () => proxyApi.getBalance(),
+    getCollateral: (params?: { amount?: Cbor }) => proxyApi.getCollateral(params),
+    getUsedAddresses: (paginate?: Paginate) => proxyApi.getUsedAddresses(paginate),
+    getUnusedAddresses: () => proxyApi.getUnusedAddresses(),
+    getChangeAddress: () => proxyApi.getChangeAddress(),
+    getRewardAddresses: () => proxyApi.getRewardAddresses(),
```

Comment:
Might as well get rid of the double proxy
```suggestion
    getRewardAddresses: () => proxyApi.getRewardAddresses(),
    experimental: {
      getCollateral: (params?: { amount?: Cbor }) => proxyApi.getCollateral(params)
    }
```